scene/gui/popup.cpp

```diff
@@ -112,6 +112,14 @@ void Popup::set_as_minsize() {
 	set_size(get_contents_minimum_size());
 }
 
+void Popup::set_close_on_parent_focus(bool p_close) {
+	close_on_parent_focus = p_close;
+}
+
+bool Popup::get_close_on_parent_focus() {
+	return close_on_parent_focus;
+}
```

Comment:
Should these be bound to scripting languages? (And if so, with a matching `close_on_parent_focus` property?)

Reply:
Bump.